client/src/main.rs

```diff
@@ -41,13 +40,10 @@ use codec::{Decode, Encode};
 use log::*;
 use my_node_runtime::{
     pallet_substratee_registry::{Enclave, Request},
-    AccountId, BalancesCall, Call, Event, Hash, Signature,
+    AccountId, BalancesCall, Call, Event, Hash,
```

Comment:
Should this then be a separate issue, since it's not really related to the RPC API issue ? (all that has changed here, is the removal of a unused reference to 'Signature' ).

Reply:
Sure, It can be a separate issue.